qa/integration-tests/src/test/java/io/zeebe/broker/it/gateway/GatewayLivenessProbeIntegrationTest.java

```diff
@@ -63,17 +56,25 @@ public void shouldReportLivenessUpIfConnectedToBroker() {
             .build();
 
     // --- when + then ---------------------------------------
-    given().spec(gatewayServerSpec).when().get(PATH_LIVENESS_PROBE).then().statusCode(200);
+    // most of the liveness probes use a delayed health indicator which is scheduled at a fixed
+    // rate of 5 seconds, so it may take up to that and a bit more in the worst case once the
+    // gateway finds the broker
+    Awaitility.await("wait until status turns UP")
+        .atMost(Duration.ofSeconds(10))
+        .pollInterval(Duration.ofMillis(100))
+        .untilAsserted(
+            () ->
+                given()
+                    .spec(gatewayServerSpec)
+                    .when()
+                    .get(PATH_LIVENESS_PROBE)
+                    .then()
+                    .statusCode(200));
```

Comment:
As discussed, please either add a timeout on the request or try once more if this assertion failed, because it could be that the request just took very long the first time and ends up with a non 200 value.